src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -22,6 +22,15 @@ if (__DEV__) {
     dangerouslySetInnerHTML: true,
     key: true,
     ref: true,
+
+    defaultValue: true,
+    valueLink: true,
+    defaultChecked: true,
+    checkedLink: true,
+    innerHTML: true,
+    suppressContentEditableWarning: true,
+    onFocusIn: true,
+    onFocusOut: true,
```

Comment:
@jimfb can you followup and add a comment in the code about this? This inline discussion will be lost in the sands of time.